responses/rental_cancelation_policies/public-scope.json

```diff
@@ -0,0 +1,23 @@
+{
+  "links": {
+    "rental_cancelation_policies.rental": "https://www.bookingsync.com/api/v3/rentals/{rental_cancelation_policies.rental}",
+    "rental_cancelation_policies.rental_cancelation_policy_items": "https://www.bookingsync.com/api/v3/rental_cancelation_policy_items/{rental_cancelation_policies.rental_cancelation_policy_items}"
+  },
+  "rental_cancelation_policies": [
+    {
+      "links": {
+        "rental": 1,
+        "rental_cancelation_policy_items": [
+          1,
+          2
+        ]
+      },
```

Comment:
it returns also `body` which seems to be the body translation in default locale, so sth like
`"body": "CANCELLATIONS. \nAny cancellation must be sent by registered letter, on strength of postmark."`
should be added